CRM/Remotetools/Upgrader.php

```diff
@@ -52,7 +52,7 @@ public function install() {
           'value'           => 'remote_contact',
           'is_reserved'     => 1,
           'description'     => E::ts(
-                'Used by the RemoteTools extension to map CiviCRM contacts to remote users oder contacts.'
+                'Used by the Remote Tools extension to map CiviCRM contacts to remote users or contacts.'
```

Comment:
Let's use *CiviRemote* as extension/framework name consistently in the UI, the user should not have to know that there are *developer tools* for this framework.